pkg/utils/utils.go

```diff
@@ -63,3 +64,11 @@ func TrimTrailingNewline(str string) string {
 	}
 	return str
 }
+
+// GetProjectRoot returns the path to the root of the project. Only to be used
+// in testing contexts, as with binaries it's unlikely this path will exist on
+// the machine
+func GetProjectRoot() string {
+	gp := os.Getenv("GOPATH")
+	return path.Join(gp, "src/github.com/jesseduffield/lazygit")
```

Comment:
You should probably take into account forked repos.

Reply:
Good point. Off the top of my head the simplest way to do that would be to traverse up till you get to 'github.com' then down two directories. Does that sound reasonable?

Reply:
wait a minute that makes no sense we just need to traverse up till we get to lazygit haha

Reply:
please see https://github.com/jesseduffield/lazygit/commit/81b07daa01244b7f0326adeba2a5429516f2d85d